README.md

```diff
@@ -209,17 +209,21 @@ individual parameters of each preference composable to achieve the functionality
 something is missing, please create an Issue so we can discuss possible solutions.
 
 # Download
-
-In your `settings.gradle` file add the following
+Add `jitpack.io` repository to your root `build.gradle` at the end of repositories:
 
 ``` groovy
-maven { url "https://jitpack.io" }
+allprojects {
+    repositories {
+        ...
+        maven { url 'https://jitpack.io' }
+    }
+}
 ```
```

Comment:
In the future, please keep changes which aren't related to separate commits. So one commit for small improvements like these and another for the build fixes. One PR for both is fine.